### ISSUE TITLE
The `packaging` library is missing in the produced `.pyz` file

### ISSUE THREAD

**Issue by sinoroc** [facundobatista/pyempaq#55]
After packaging a simple project without dependencies I tried to run it and received this error message:

```
$ python3.11 Something.pyz 
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/tmp/tmp.EQhCLfHNpy/Something.pyz/__main__.py", line 20, in <module>
ModuleNotFoundError: No module named 'packaging'
```

It seems to me like the same way [`appdirs` is installed](https://github.com/facundobatista/pyempaq/blob/48f54a5f38d1c5c1e3dd413132be79027e458a84/pyempaq/main.py#L211) in the `venv` directory inside the `.pyz` output file, `packaging` should be installed as well.

Also I think you should consider migrating from *appdirs* to [*platformdirs*](https://pypi.org/project/platformdirs/), which is more up-to-date (as far as I know, it is vendored-in by *pip* itself and thus will keep receiving a lot of attention in terms of continuous maintenance).

**Comment by facundobatista** [comment]
Thanks for the report. 

The main issue is already fixed in #56 . For the other suggestion I opened a clean issue in #57.

Thanks again!

**Comment by facundobatista** [comment]
FTR, this is already released as part of 0.3.1

**Comment by sinoroc** [comment]
[You asked](https://discuss.python.org/t/announcement-pyempaq-0-3-released/26016/10) "*in which platform/environment were you running the `.pyz` that didn’t find `packaging`?*"

It was with *Python 3.11* on *Ubuntu Linux 18.04*. As far as I know there is no reason why *packaging* should have been available, it is not part of the standard library.
